### PR TITLE
Stage8 seed config state

### PR DESCRIPTION
**Pull request by Peartes**
Make sure stage8 can access the whitelist after it has been cleared in stage7 and display the whitelist in a nice format. Closes this [issue](https://www.notion.so/curvelabs/339a249185e14ddcbd731e5ba07c46c7?v=f7cba822bb9043a5a87ab80b5e2b40ef&p=fe9141be3ed046e78d524b9aa3fbd255)

### CODE REVIEW

File: src/newSeed/baseStage.scss [hunk 1]
```diff
@@ -545,6 +545,7 @@
   }
 
   &.stage8 {
+    max-width: 90vw;
```
Comment (dkent600) [inline src/newSeed/baseStage.scss:548]: This is not part of the card.  please remove.  Create a new card if needed.

File: src/newSeed/stage8.html [hunk 2]
```diff
@@ -19,7 +19,8 @@
           ${wizardState.seedTokenSymbol} to the Seed contract.
           This amount includes any service fees.
         </p>
-        <p><etherscanlink address.to-view="wizardState.seedAdminAddress" text.to-view="wizardState.seedAdminAddress | smallHexString"></etherscanlink> is the sole administrator of the new Seed<span if.to-view="!!seedConfig.seedDetails.whitelist"> and are responsible for managing its whitelist</span>.</p>
+        <p><etherscanlink address.to-view="wizardState.seedAdminAddress" text.to-view="wizardState.seedAdminAddress | smallHexString"></etherscanlink> is the sole administrator of the new Seed and 
+        <div if.to-view="!!wizardState.whiteList" class="ellipses"><a href="${wizardState.whiteList}" target="_blank" rel="noopener noreferrer">${wizardState.whiteList}</a> are responsible for managing its whitelist.</div></p>
```
Comment (dkent600) [inline src/newSeed/stage8.html:23]: why did you change this to a div?

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/newSeed/stage8.html [hunk 3]
```diff
@@ -19,7 +19,7 @@
           ${wizardState.seedTokenSymbol} to the Seed contract.
           This amount includes any service fees.
         </p>
-        <p><etherscanlink address.to-view="wizardState.seedAdminAddress" text.to-view="wizardState.seedAdminAddress | smallHexString"></etherscanlink> is the sole administrator of the new Seed<span if.to-view="!!seedConfig.seedDetails.whitelist"> and are responsible for managing its whitelist</span>.</p>
+        <p><etherscanlink address.to-view="wizardState.seedAdminAddress" text.to-view="wizardState.seedAdminAddress | smallHexString"></etherscanlink> is the sole administrator of the new Seed<span if.to-view="!!wizardState.whiteList"> and <a href="${wizardState.whiteList}" target="_blank" rel="noopener noreferrer">${wizardState.whiteList} </a> are responsible for managing its whitelist</span>.</p>
```
Comment (dkent600) [inline src/newSeed/stage8.html:22]: why the extra "`<a href="${wizardState.whiteList}" target="_blank" rel="noopener noreferrer">${wizardState.whiteList} </a>`"?  Have you tested this?

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/newSeed/stage8.html (hunk 3 above)
Comment (Peartes) [reply src/newSeed/stage8.html:22]: I thought the whitelist address was to be displayed

Comment (dkent600) [reply src/newSeed/stage8.html:22]: Unless someone is asking for it, then please don't add it.  If they are asking for it, then we need to wording of the sentence to make sense.

In any case, "and are responsible" should be "and is responsible".

**Review by dkent600 — CHANGES_REQUESTED** [review]

**Review by dkent600 — APPROVED** [review]
